package.json

```diff
@@ -22,8 +22,8 @@
     "registry": "https://registry.npmjs.org/"
   },
   "scripts": {
-    "test": "asp --verbose",
-    "test:ci": "asp --summary",
+    "test:asp": "asp --verbose",
```

Comment:
what the point of adding ":asp" ?

Reply:
because in this repo, we have tests written for as-pect test framework and the new NFT contract test is written for envy test framework.

Reply:
I will let you create a new issue for this